hooks/pre-command

```diff
@@ -15,7 +15,7 @@ if [[ "${!DEBUG_FLAG:-false}" =~ (true|on|1) ]] ; then
 fi
 
 RUN_TYPE=download
-COMMIT_HASH=$(git rev-parse HEAD)
+REPO=$(plugin_read_config REPO)
```

Comment:
**question:** Could something like this be used as a default? https://buildkite.com/docs/pipelines/environment-variables#BUILDKITE_REPO but be overridable